### PR TITLE
Added title and short descr

### PR DESCRIPTION
**Pull request by stephlaf**
Please @annemano review my `code` and merge if all good 😁

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h2>This is our amazing app!</h2>
```
Comment (annemano) [inline app/views/pages/home.html.erb]: I would advise to change the `h2` to a `h1` for SEO purposes!

Comment (stephlaf) [reply app/views/pages/home.html.erb]: Oh man, missed that one. Let me correct.

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+<p>This is a description</p>
```
Comment (stephlaf) [inline app/views/pages/home.html.erb:2]: Is this good with you though?

Comment (annemano) [reply app/views/pages/home.html.erb:2]: Yup that's fine! :)

**Comment by stephlaf** [comment]
Hey @annemano please review

**Review by annemano — APPROVED** [review]
LGTM! 🚀

**Pull request merged**